src/features/farming/hud/components/PlaceableController.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { useContext } from "react";
+import { useActor } from "@xstate/react";
+import { Button } from "components/ui/Button";
+import { OuterPanel } from "components/ui/Panel";
+import { MachineInterpreter } from "features/game/expansion/placeable/editingMachine";
+import { Context } from "features/game/GameProvider";
+
+import confirm from "assets/icons/confirm.png";
+import cancel from "assets/icons/cancel.png";
+
+export const PlaceableController: React.FC = () => {
+  const { gameService } = useContext(Context);
+  const child = gameService.state.children.editing as MachineInterpreter;
+
+  const [
+    {
+      context: { collisionDetected },
+    },
+    send,
+  ] = useActor(child);
+
+  const handleConfirmPlacement = () => {
+    send("PLACE");
+  };
+
+  const handleCancelPlacement = () => {
+    send("CANCEL");
+  };
+
+  return (
+    <div className="fixed bottom-2 left-1/2 -translate-x-1/2">
```

Comment:
Consider adding some higher z-indexes to this for small mobiles.

<img width="447" alt="Screen Shot 2022-08-11 at 1 05 05 pm" src="https://user-images.githubusercontent.com/11745561/184058503-9af439c0-d92d-4597-91ff-3daf4bff026c.png">

Reply:
I'm wondering, should a user even be able to click the build hammer button icon?

It may be worth hiding the entire HUD while in edit mode